desc/backend.py

```diff
@@ -112,6 +113,28 @@ def put(arr, inds, vals):
             return arr
         return jnp.asarray(arr).at[inds].set(vals)
 
+    def set_default_cpu(func):
```

Comment:
I feel like we could come up with a better/more descriptive name, `default_device_cpu`? or something? but not a dealbreaker